test/e2e/logforwarding/syslog/forward_to_syslog_test.go

```diff
@@ -139,6 +143,17 @@ var _ = Describe("[ClusterLogForwarder] Forwards logs", func() {
 				Expect(logStore.GrepLogs(grepmsgid, helpers.DefaultWaitForLogsTimeout)).To(Equal(expectedMsgID), "Expected: "+expectedMsgID)
 				expectedProcID := forwarder.Spec.Outputs[0].Syslog.ProcID
 				Expect(logStore.GrepLogs(grepprocid, helpers.DefaultWaitForLogsTimeout)).To(Equal(expectedProcID), "Expected: "+expectedProcID)
+				grepMsgContent := fmt.Sprintf(`grep %s %%s| grep pod_name | tail -n 1 | sed -n -e 's/^.*openshift://p'`, logGenPod)
+				str, err := logStore.GrepLogs(grepMsgContent, helpers.DefaultWaitForLogsTimeout)
+				Expect(err).To(BeNil())
+				str = strings.ReplaceAll(str, "=>", ":")
+				msg := map[string]map[string]string{}
+				err = json.Unmarshal([]byte(str), &msg)
+				Expect(err).To(BeNil())
+				expectedLabelSecure := forwarder.Spec.Pipelines[0].Labels["secure"]
```

Comment:
This is part of the setup which you already know that could be defined initially as a `const`.  Doing so earlier in the test will allow you to remove these lines making the test more clear.  Comment applies to all these labels

Reply:
Do you mean to define all the "expected" variables as constants above?

Reply:
reduce code duplication as much as we can where it makes sense